pkg/openSearch/openSearchQuery/compareHandler.go

```diff
@@ -187,6 +187,51 @@ func createTermQuery(fieldName string, fieldValue any, fieldKeys []string, query
 	}
 }
 
+func HandleCompareOperatorIsGreaterThanRating(fieldName string, fieldKeys []string, fieldValue any, querySettings *QuerySettings) esquery.Mappable {
+	ratingRange := getStringRange(fieldName, fieldValue.(string), querySettings)
+	return esquery.Range(fieldName).
+		Gt(ratingRange.Max)
+}
+
+func HandleCompareOperatorIsLessThanRating(fieldName string, fieldKeys []string, fieldValue any, querySettings *QuerySettings) esquery.Mappable {
+	ratingRange := getStringRange(fieldName, fieldValue.(string), querySettings)
+	return esquery.Range(fieldName).
+		Lt(ratingRange.Min)
+}
+
+func HandleCompareOperatorIsGreaterThanOrEqualToRating(fieldName string, fieldKeys []string,
+	fieldValue any, querySettings *QuerySettings,
+) esquery.Mappable {
+	ratingRange := getStringRange(fieldName, fieldValue.(string), querySettings)
+	return esquery.Range(fieldName).
+		Gte(ratingRange.Max)
```

Comment:
```suggestion
		Gte(ratingRange.Min)
```
Example: `4 >= 'medium'` (which has range [4,7]) should be true true.

Reply:
thanks - nice spotting